validator/src/main/java/com/google/daq/mqtt/sequencer/sequences/ConfigSequences.java

```diff
@@ -51,7 +51,7 @@ public class ConfigSequences extends SequenceBase {
   private static final Duration CONFIG_QUERY_INTERVAL = Duration.ofSeconds(30);
 
   @Test(timeout = ONE_MINUTE_MS)
-  @Feature(stage = STABLE, bucket = SYSTEM, nostate = true)
+  @Feature(stage = STABLE, bucket = SYSTEM)
```

Comment:
Likely needs a comment, but this "nostate = true" is intentional since I don't want things marked as "nostate" to just blindly pass -- I wanted it on record that they still essentially fail.  Was there a specific reason you needed this, or was it just because it seemed like a good idea (which is a perfectly reasonable assumption!)?

Reply:
It seems either test declaration or expected device behavior is not correct me. It is expecting state message from the device. But it make sense to fail if it is intentional.

Reply:
I talked w/ Nour about this and we think you're right and I'll return it to the way you had it -- I had done it that way because of development testing but it doesn't make sense to keep it like that.  I'll do that in another PR since it really doesn't have anything to do with your changes and requires some other changes to make it pass CI testing.